tests/processes/ProcessingTest.py

```diff
@@ -566,8 +567,8 @@ def test_yield_ignore_ranges(self):
     def test_loaded_bears_with_error_result(self):
         class BearWithMissingPrerequisites(Bear):
 
-            def __init__(self, section, queue, timeout=0.1, debugger=False):
-                Bear.__init__(self, section, queue, timeout, debugger)
+            def __init__(self, *args, **kwargs):
+                Bear.__init__(self, *args, **kwargs)
```

Comment:
these changes are already reflected in Profiler interface PR once these changes gets merge there I will rebase (the purpose of reflecting these changes here is preventing the coverage and test failure problem)